sql/hive/src/test/scala/org/apache/spark/sql/hive/CachedTableSuite.scala

```diff
@@ -203,4 +203,18 @@ class CachedTableSuite extends QueryTest {
     sql("DROP TABLE refreshTable")
     Utils.deleteRecursively(tempPath)
   }
+
+  test("SPARK-10327 Cache Table is not working while subquery has alias in its project list") {
```

Comment:
We are getting rid of test data, and I think generally all tests should just define a dataframe inline with the test.